pkg/justification/processor.go

```diff
@@ -40,6 +42,7 @@ import (
 type Processor struct {
 	jvspb.UnimplementedJVSServiceServer
 	kms         *kms.KeyManagementClient
+	fsClient    *firestore.Client
```

Comment:
We should abstract the implementation away for better encapsulation. So instead of passing in a firestore client, I suggest passing in a `RemoteConfig` (to be defined) interface. The actual implementation will be firestore based.